ansible/roles/airflow/files/docker-trampoline

```diff
@@ -97,7 +98,7 @@ case "$script" in
     volargs="${volargs} --env-file=/etc/af-worker/s3read.env"
     volargs="${volargs} --volume=$public/sanitised-s3-ls:$public/sanitised-s3-ls:rw"
     script=""
-    set -- /usr/local/bin/aws_s3_ls.py --url "s3://ooni-public/sanitised/$bucket" --s3-ls "$public/sanitised-s3-ls/${bucket}.json.gz"
+    set -- /usr/local/bin/aws_s3_ls.py --url "s3://ooni-data/sanitised/$bucket" --s3-ls "$public/sanitised-s3-ls/${bucket}.json.gz"
```

Comment:
NB: publishing of `sanitised` files is not fixed yet. Maybe it never will, something is triggering the bug in luigid that prevents that pipeline from progressing on 2017-10-09 for 36h+ I don't feel like that the investigation is well-spent time.